### ISSUE TITLE
Cross members not fully attached

### ISSUE THREAD

**Issue by kfrancis** [GliaX/faceshield#25]
![2020-03-27 png (1811×1039) 2020-03-27 12 26 28](https://user-images.githubusercontent.com/67574/77777655-3ea09180-7026-11ea-8086-5581beae6e25.png)

**Comment by mza1979m** [comment]
Yes, I imported the model into Fusion 360 and noticed this as well, so I fixed it, and while I was at it I reinforced the center section that goes directly against the forehead, as it is was quite flimsy, by doubling it up. I'd share the new version, but I'm not sure how this works. I'm familiar with the overall concept of "Open-Source", but not sure about the details, updating models, sharing, etc...
